src/main/java/mate/controller/GetAllCarController.java

```diff
@@ -0,0 +1,27 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet("/cars")
+public class GetAllCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService)
+            injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Car> cars = carService.getAll();
+        req.setAttribute("cars", cars);
+        req.getRequestDispatcher("/WEB-INF/views/AllCars.jsp").forward(req, resp);
```

Comment:
please create the appropriate folders for jsp pages (`cars`, `drivers`, `manufactures`) and divide jsp pages between appropriate folders:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/all.jsp").forward(req, resp);
```
also simplify the `jsp` files names use `add.jsp`, `all.jsp` etc.